swarm/network/stream/intervals_test.go

```diff
@@ -48,7 +48,7 @@ func newIntervalsStreamerService(ctx *adapters.ServiceContext) (node.Service, er
 	kad := network.NewKademlia(addr.Over(), network.NewKadParams())
 	store := stores[id].(*storage.LocalStore)
 	db := storage.NewDBAPI(store)
-	delivery := NewDelivery(kad, db)
+	delivery := NewDelivery(kad, db, nil)
```

Comment:
Am I the only one not liking unnamed parameters to functions? e,g. like `nil`, `true`,`false`? It makes it hard to follow what's going on and I have noticed can be the cause of easily missed issues, as simply swapping `true` for `false` sometimes can have important results.

Of course, this is also not strictly related to this PR, is off-topic and is a general issue. But I wanted to bring it up once at least, for now.

Reply:
We discussed this in a comment in this PR further above. Let's discuss during the roundtable next week, we agreed this is out-of-scope for the PR, but it is not just you who finds this confusing and more complicated than necessary.